cmd/vm.go

```diff
@@ -88,18 +88,6 @@ func saveKeyPair(keyPairs *egoscale.SSHKeyPair, vmID egoscale.UUID) {
 	}
 }
 
-func deleteKeyPair(vmID egoscale.UUID) error {
-	folder := getKeyPairPath(vmID.String())
-
-	if _, err := os.Stat(folder); !os.IsNotExist(err) {
-		if err := os.RemoveAll(folder); err != nil {
-			return fmt.Errorf("the SSH private key could not be deleted: %s", err)
-		}
-	}
-
-	return nil
-}
-
```

Comment:
Dead code, right?

Reply:
Yes, used only by the `exo lab kube` commands.